src/bin/cratesfyi.rs

```diff
@@ -12,12 +12,29 @@ use docs_rs::{
     Storage,
 };
 use once_cell::sync::OnceCell;
+use sentry_anyhow::capture_anyhow;
+use sentry_log::SentryLogger;
 use structopt::StructOpt;
 use strum::VariantNames;
 
 pub fn main() {
     let _ = dotenv::dotenv();
-    logger_init();
+
+    let _sentry_guard = if let Ok(sentry_dsn) = env::var("SENTRY_DSN") {
+        rustwide::logging::init_with(SentryLogger::with_dest(logger_init()));
+        Some(sentry::init((
+            sentry_dsn,
+            sentry::ClientOptions {
+                release: Some(docs_rs::BUILD_VERSION.into()),
+                attach_stacktrace: true,
```

Comment:
Hmm, why set this? The docs say:
> When enabled, stack traces are automatically attached to all messages logged. Stack traces are always attached to exceptions; however, when this option is set, stack traces are also sent with messages. This option, for instance, means that stack traces appear next to all log messages.

and I don't think it's super useful to show a backtrace for the top-level error! macro, since it will always be the same.

Reply:
I wanted to have automatic stack-traces for `log::error!` calls. 

I might be wrong, but in testing the stacktrace looked correct, from the place where the `log::error` call was made: 

<img width="679" alt="grafik" src="https://user-images.githubusercontent.com/540890/132998527-ce829652-b5d4-4369-84c7-c3e9cce4a308.png">

( I had an additional `log::error` call in `main`)

Reply:
> I might be wrong, but in testing the stacktrace looked correct, from the place where the log::error call was made:

Right - but my point is that usually we want the stacktrace of the error itself, not the error! call.

I guess that's an argument for getting rid of error! altogether and changing all those to `capture_anyhow`; I would be ok leaving this in until we do that.